wx/tools/wxget_docs_demo.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#----------------------------------------------------------------------
+# Name:        wx.tools.wxget_docs_demo
+# Purpose:     wx fetch & launch demo or docs
+#
+# Author:      Steve Barnes
+#
+# Created:     06-Aug-2017
+# Copyright:   (c) 2017 by Steve Barnes
+# Licence:     wxWindows license
+# Tags:        phoenix-port, py3-port
+#
+# Module to allow the correct version of the documents &/or demos to be
+# launched after, if necessarily being fetched.
+#----------------------------------------------------------------------
+
+"""
+wxget_docs_demo.py -- Launch the appropriate wx Docs or Demo.
+
+Usage:
+    wxget_docs_demo [docs|demo]
+
+Will install if missing, the requested item for the current version and then
+launch it.
+"""
+from __future__ import (division, absolute_import, print_function, unicode_literals)
+
+import sys
+import os
+import subprocess
+import webbrowser
+import tarfile
+if sys.version_info >= (3,):
+    from urllib.error import HTTPError
+    import urllib.request as urllib2
+    import urllib.parse as urlparse
+    from urllib.request import pathname2url
+else:
+    import urllib2
+    from urllib2 import HTTPError
+    import urlparse
+    from urllib import pathname2url
+
+import wx
+
+import wxget
+
+APP = None
+WXVERSION = wx.version().split(' ')[0]
+MAJOR = WXVERSION.split('.')[0]
+if MAJOR != '4':
+    raise ValueError("wx Versions other than 4 not supported!")
```

Comment:
`wx.VERSION` is a tuple containing the version number components. Using it will save a couple lines here. Also, there is `wx.VERSION_STRING` for the string version of the full version number.